crates/cansat-stm32f4/src/startup.rs

```diff
@@ -95,20 +96,18 @@ pub fn init_drivers(mut board: Board, statik: &'static mut Statik) -> Result<Can
         .tap_err(|e| defmt::error!("Failed to initialize SD logger: {}", e))
         .ok();
 
-    // TODO: init lora here
+    let lora = init_lora(board.serial6)
+        .tap_err(|e| defmt::error!("Failed to initialize Lora: {}", defmt::Debug2Format(&e)))
```

Comment:
Implement `defmt::Format` for `cansat_lora::Error`, and then
```suggestion
        .tap_err(|e| defmt::error!("Failed to initialize Lora: {}", &e))
```